src/Button.js

```diff
@@ -74,7 +85,13 @@ class AriaMenuButtonButton extends React.Component {
       onBlur: this.context.ambManager.handleBlur
     };
 
-    specialAssign(buttonProps, props, checkedProps);
+    const reserved = Object.assign({}, checkedProps);
```

Comment:
I'd like to avoid `Object.assign` so IE-supporting users don't need a polyfill. `specialAssign()` should work, right?

Reply:
I need to remove the `disabled` attribute for the `checkedProps` and I guess it's a bad idea to change the actual object for that. I'll just create a new object with only the `tag` and `children` props attached to it.